src/styles/blocks/card.scss

```
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+
+  &__image {
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    margin: 0;
+    margin-top: 40px;
+
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    align-self: start;
+
+    margin-top: 4px;
+
+    color: $secondary-color;
+  }
+
+  &__raiting {
+    position: relative;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+  }
+
+  &__reviews {
+    padding-top: 18px;
+    align-self: flex-end;
+  }
+
+  &__price {
+    display: flex;
+    align-self: flex-start;
+    justify-content: space-between;
+
+    margin-top: 24px;
+
+    width: 100%;
+  }
```

```suggestion
  &__price {
    display: flex;
    align-self: flex-start;
    justify-content: space-between;
    margin-top: 24px;
    width: 100%;
  }
```

Go though your code and remove all unneeded empty lines